src/sage/combinat/permutation.py

```diff
@@ -5339,18 +5342,18 @@ def merging_cycles(list_of_cycles):
                     gamma = Permutation(tuple(new_Perm))
                     yield gamma
 
-        def rewind(L, k):
+        def rewind(L, n):
             """
-            Construct the list M such that M[(j*k)%(len(M))] == L[j].
+            Construct the list M such that M[(j*n)%(len(M))] == L[j].
             """
             M = [0 for _ in L]
             m = len(M)
             for j in range(m):
-                M[(j*k)%m] = L[j]
+                M[(j*n)%m] = L[j]
```

Comment:
The linter will require this to be written as `M[(j * k) % m]`.

Reply:
I don't think so. Plus PEP8 says we can put higher precedence operators without spaces (and `[]` counts as an operator). I don't think the linter is (nor should be) that strict.